uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/sub-controllers/request-modal-details-toil.controller.js

```diff
@@ -412,6 +416,16 @@ define([
       }
     }
 
+    /**
+     * Opens the CRM modal that allows to edit TOIL in days amounts options
+     * and reloads these options in the Leave Request Modal
+     * if they are changed via the CRM modal
+     */
+    function openToilInDaysAccrualOptionsEditor () {
+      crmAngService.loadForm('/civicrm/admin/options/hrleaveandabsences_toil_amounts?reset=1')
+        .on('crmFormSuccess', loadToilAmounts);
```

Comment:
what about passing a cache parameter so we can still benefit from cached toil amounts?

```js
function openToilInDaysAccrualOptionsEditor () {
  crmAngService.loadForm('/civicrm/admin/options/hrleaveandabsences_toil_amounts?reset=1')
    .on('crmFormSuccess', function () {
      // false means no cache:
      loadToilAmounts(false);
    });
}

function loadToilAmounts (cache) {
  // ...
}
```

Reply:
Done.

Reply:
Not done! If we do that, then if we change options, then close modal, then reopen modal - it will should previously cached results. We need to somehow **flush** the cache when options are changed via wrench.

cc @AkA84

Reply:
What if, when you close the modal:
* You flush the $http cache (https://stackoverflow.com/questions/31168708/invalidating-all-http-caches-in-angular, first thing i found)
* Reload the options

Of course there is a bit of difference between making it work and making it work and having it implemented in a clean way (for example I wouldn't want a controller to flush the cached data related to an api service), but let's see first if it works

Reply:
@AkA84 When you say "Modal" which modal do you refer to: editor or leave request modal? I think that it would be actually better to have a following logic:

1) If you load something with cache enabled - leave it as it is now
2) If you load something with cache disabled - flush the cache (new functionality)

cc @reneolivo